src/files/files.ts

```diff
@@ -105,20 +105,30 @@ async function handleZippedModelFiles<T extends Content>(
     fileFormat = 'arraybuffer'
   }
 
+  let baseDir = ''
+
   zip.forEach((filePath, file) => {
     if (
       !basename(filePath).startsWith('.') &&
-      basename(filePath) !== WEARABLE_MANIFEST &&
-      basename(filePath) !== BUILDER_MANIFEST &&
-      basename(filePath) !== SCENE_MANIFEST
+      ![WEARABLE_MANIFEST, BUILDER_MANIFEST, SCENE_MANIFEST].includes(
+        basename(filePath)
+      )
     ) {
       fileNames.push(filePath)
       promiseOfFileContents.push(file.async(fileFormat) as Promise<T>)
+    } else if (
+      [WEARABLE_MANIFEST, SCENE_MANIFEST].includes(basename(filePath))
+    ) {
+      baseDir = dirname(filePath) === '.' ? '' : `${dirname(filePath)}/`
     }
   })
 
   const fileContents = await Promise.all(promiseOfFileContents)
   const content: RawContent<T> = fileNames.reduce((acc, fileName, index) => {
+    if (fileName.endsWith('/')) {
+      return acc
+    }
```

Comment:
When would a filename end with `/`?
Is the library listing directories?

Reply:
> Is the library listing directories?
yeap, it's the library listing direcotries